notebooks/tm/hospitals-isochrones/02-contours-to-adminlevel.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.            brgy_population_count[f"{time_col}_pop_perc_reached"] = (
</pre><p>change to f"pct_pop_reached_{HEALTH_FACILITY_TYPE}_{time_col}"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Did not follow this since just followed the current ERD and also we'll have separate tables for them as per EM's request.